src/main/java/uk/gov/justice/digital/delius/data/api/Enforcement.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Enforcement {
+    @NotNull
+    @ApiModelProperty(name = "EnforcementAction", required = true)
+    private KeyValue enforcementAction;
+}
```

Comment:
is this really all we need here? none of the dates?

Reply:
We are only showing enforcement action description, so don't need anything else.